.github/workflows/standard_spa_pr_NPM.yml

```diff
@@ -372,7 +372,9 @@ jobs:
         node: [18]
     concurrency:
       group: run-e2e-tests-${{ needs.create-shared-vars.outputs.groupId }}-${{ matrix.containers }}
-      cancel-in-progress: true
+      # To avoid the following error, we leave this disabled: Canceling since a higher priority waiting request for 'run-e2e-tests-PR-web-settings-91-1' exists
```

Comment:
Addresses the following reason for a timeout `Canceling since a higher priority waiting request for 'run-e2e-tests-PR-web-... exists`